{
	"id": "oval:org.debian:def:146718536375278291597013671721184100806",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2021-21330 python-aiohttp",
		"affected": {
			"family": "unix",
			"platform": "Debian GNU/Linux 10",
			"product": "python-aiohttp"
		},
		"reference": [
			{
				"source": "CVE",
				"ref_id": "CVE-2021-21330",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-21330"
			}
		],
		"description": "security update",
		"debian": {
			"moreinfo": "no info",
			"dsa": "DSA-4864"
		}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterias": [
					{
						"operator": "AND",
						"criterions": [
							{
								"test_ref": "oval:org.debian.oval:tst:2",
								"comment": "all architecture"
							},
							{
								"test_ref": "oval:org.debian.oval:tst:20801",
								"comment": "python-aiohttp DPKG is earlier than 3.5.1-1+deb10u1"
							}
						]
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.debian.oval:tst:1",
				"comment": "Debian 10 is installed"
			}
		]
	}
}
